start-site/src/main/resources/application.yml

```diff
@@ -1253,7 +1255,7 @@ initializr:
           artifactId: spring-cloud-stream-reactive
     - name: Pivotal Cloud Foundry
       bom: spring-cloud-services
-      compatibilityRange: "[2.0.0.RELEASE,2.3.0.M1)"
+      compatibilityRange: "[2.0.0.RELEASE,2.3.x.RELEASE)"
```

Comment:
This should be rather `2.4.0-M1` to indicate that the range ends at the first milestone of the `2.4.x` line (excluded).